crates/chia-consensus/src/spendbundle_validation.rs

```diff
@@ -33,3 +33,41 @@ pub fn get_flags_for_height_and_constants(height: u32, constants: &ConsensusCons
     }
     flags
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::consensus_constants::TEST_CONSTANTS;
+
+    #[test]
+    fn test_get_flags() {
+        assert_eq!(
+            get_flags_for_height_and_constants(
+                TEST_CONSTANTS.soft_fork4_height - 1,
+                &TEST_CONSTANTS
+            ),
+            33554592
```

Comment:
let's use the named flags here instead. It's really hard to tell whether this is correct or not by just reading the test

Reply:
Done

Reply:
I don't see this change